Quoridor.Model/Position.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Quoridor.Model
+{
+    class Position
+    {
+        public Position(int x, int y) { }
```

Comment:
Maybe there should be fields for x and y?

Reply:
+